code/addons/themes/README.md

```diff
@@ -0,0 +1,72 @@
+# `@storybook/addon-themes
+
+Storybook Addon Themes can be used which between multiple themes for components inside the preview in [Storybook](https://storybook.js.org).
+
+![React Storybook Screenshot](https://user-images.githubusercontent.com/42671/98158421-dada2300-1ea8-11eb-8619-af1e7018e1ec.png)
```

Comment:
I assume this is not the screenshot you want to use?

Reply:
No, Joao is going to help me out with an optimized video like we have for the other addons